docs/configuration.md

```diff
@@ -2013,7 +2013,62 @@ Apart from these, the following properties are also available, and may be useful
     </tr>
 </table>
 
+### HTTP Security Headers
```

Comment:
I think this could move to `security.md` as a security related advanced configurations.